js/plugins/mcp/README.md

```diff
@@ -13,53 +15,94 @@ To get started, you'll need Genkit and the MCP plugin:
 npm i genkit genkitx-mcp
 ```
 
-## MCP Client
+## MCP Client Manager
 
-To create an MCP client, you call the `mcpClient` function to generate a Genkit plugin for an MCP server. For example, to use MCP's example [filesystem server](https://github.com/modelcontextprotocol/servers/tree/main/src/filesystem):
+To connect to one or more MCP servers, you use the `createMcpClientManager` function. This function returns a Genkit plugin that manages connections to the configured MCP servers.
 
 ```ts
 import { genkit } from 'genkit';
-import { mcpClient } from 'genkitx-mcp';
+import { createMcpClientManager } from 'genkitx-mcp';
 
-// the filesystem server requires one or more allowed directories
+// Example: Configure a client manager for a local filesystem server
+// and a hypothetical remote Git server.
 const ALLOWED_DIRS = ['/Users/yourusername/Desktop'];
 
-const filesystemClient = mcpClient({
-  name: 'filesystem',
-  serverProcess: {
-    command: 'npx',
-    args: ['-y', '@modelcontextprotocol/server-everything', ...ALLOWED_DIRS],
+const mcpManager = createMcpClientManager({
+  name: 'myMcpClients', // A name for the manager plugin itself
+  mcpClients: {
+    // Each key (e.g., 'fs', 'git') becomes a namespace for the server's tools.
+    fs: { // Configuration for the first MCP server
+      serverProcess: {
+        command: 'npx',
+        args: ['-y', '@modelcontextprotocol/server-everything', ...ALLOWED_DIRS],
+      },
+    },
+    git: { // Configuration for a second MCP server (hypothetical)
+      serverUrl: 'http://localhost:8080/mcp',
+      // disabled: true, // Optionally disable a server
+    },
   },
 });
 
 const ai = genkit({
   plugins: [
-    filesystemClient /* ... other plugins such as model providers ...*/,
+    /* ... install plugins such as model providers ...*/
   ],
 });
+
+const mcpTools = await mcpManager.getAllTools(ai);
```

Comment:
Let's go with `getActiveTools` since it shouldn't return tools from disabled/inactive servers.